### ISSUE TITLE
Feature Request: Viewing Options

### ISSUE THREAD

**Issue by TobyBaril** [AliView/AliView#96]
Firstly, this is my favourite alignment viewer/editor, thanks for making something so simple and nice to use!

I wonder if it might be possible to add more viewing options, such as custim colouring for nucleotides/amino acids?
Also, perhaps more of a "trend" thing at the moment, but a dark mode would be really nice. I'm sitting and editing lots of alignments at the moment, and having a dark background would really help the eyes!

**Comment by andersla** [comment]
Thanks for your kind words about AliView!
Feature request noted and very valid!

**Comment by Lazy-tortuga** [comment]
I agree! A dark background feature would be better for long-hour working

**Comment by andzandz11** [comment]
I desperately need a dark mode. I have to edit 1135 alignments in 2 weeks.
